config/backup/models/backup.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+Backup::Model.new(:intercity, 'Intercity full backup') do
+  ##
+  # PostgreSQL [Database]
+  #
+  database PostgreSQL do |db|
+    # To dump all databases, set `db.name = :all` (or leave blank)
+    db.name               = "intercity"
+    db.host               = "localhost"
```

Comment:
I assume you have used the default intercity-docker system. Which assumes you host on the same machine. If not, you're on your own for now. (Agreed?)